### PR TITLE
VACMS-844 Update node_link_report to reduce false failures.

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

See #844 and $856 

## Testing done
Examine the following pages which had notoriously high numbers of falsely reported broken links.
/health-care/how-to-apply
/health-care/after-you-apply
/education/transfer-post-9-11-gi-bill-benefits
/pittsburgh-health-care/staff-profiles/brenda-shaffer
/outreach-and-events/events/events-10
/pittsburgh-health-care/stories/stroke-survivortime-is-brain
/pittsburgh-health-care/research/safety-security/animal-research
/disability/va-claim-exam
/disability

## QA steps

As as an admin or content admin (the only roles who can see the node link report) visit the following pages and validate there are no fake broken links.
- [x] /health-care/how-to-apply
- [x] /health-care/after-you-apply
- [x] /education/transfer-post-9-11-gi-bill-benefits
- [x] /pittsburgh-health-care/staff-profiles/brenda-shaffer
- [x] /outreach-and-events/events/events-10
- [x] /pittsburgh-health-care/stories/stroke-survivortime-is-brain
- [x] /pittsburgh-health-care/research/safety-security/animal-research
- [x] /disability/va-claim-exam
- [x] /disability

## Definition of Done
- [ ] Product release notes 
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by kevwalsh** [comment]
nice, the redirected links are great. 

i see a lot of false positives here, for drupal-managed files: 
http://pr1014.ci.cms.va.gov/pittsburgh-health-care/research/safety-security/animal-research

Thoughts on those? 

That page also has some layout problems, i also think the printed links on that block should be allowed to break, other than just the `-`. maybe the links need a `word-break: break-all;` or something similar.

**Comment by swirtSJW** [comment]
@kevwalsh Those broken links are all to internal files which do not resolve on CI because CI does not do any file syncing right now so they legitimately 404.  They should not be broken once this hits dev, stage, prod.

The layout is a different issue.   I can try to remedy that.

**Comment by swirtSJW** [comment]
@kevwalsh I pushed an update to fix the url length breaking the layout.

File: config/sync/node_link_report.settings.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ display_on_node_view: node_view
 display_on_node_edit: node_edit
 display_on_node_preview: node_preview
 additional_domains_as_internal: www.va.gov
-domains_to_skip: ''
+domains_to_skip: "charlesajudge.com\r\nwww.instagram.com\r\nwww.linkedin.com\r\nwww.redroof.com\r\nwww.youtube.com\r\n"
 enable_reporting_skipped_links: 1
 user_agent: 'VA.gov CMS link checker'
+path_patterns_to_skip: "/block/*\r\nmedia/*/edit\r\n/node/*/edit\r\n/user/*\r\n"
+decoupled_frontend: 'https://www.va.gov'
```
Comment (kevwalsh) [inline config/sync/node_link_report.settings.yml]: nice!

**Pull request merged**